### PR TITLE
Laser Turret Resprite

### LINKED ISSUE #216: Priority missing sprites from the WALLENING MEGA LIST
image
Priority 8/10, these show up immediately in game on shuttles and arrivals.

Firelocks need a tweak for the E/W Sprites, these look the same when down and up.
image
Double, rather than single for the horizontal case?

image
Investigate painting frames. We may not have enough frame sprites?

image
Windoor animation is jumping on open, but not close.

Privacy Shutters need a new sprite, nothing too crazy here
image

image
HOPline Button Mystery. IYKYK

image

No window shutters? 😢  Rad shutters too
Bro
image

Floating buttons
image

image
Items dropped from closets and crates should be shifted up to match their spawn pixel_y.

image
Items that are randomly placed should probably use pixel_z instead of pixel_y. UPSTREAM UPSTREAM!
image
Window post explosion lost it's frame, and still smoothed with other windows. Ideally, if it's not on a frame, it should smooth, anchored or not, PLUS they shouldn't be anchorable.
image
Escape pod emergency suits needs a directional.

image

image
Wooden airlocks need directiona- oh we already have that.

image

image
Has no directionals.

image
~~Thindows smoothing issue? Something is blocking emissives.~~ handled

image
~~Need turret sprites.~~ handled

image
More dooooors

image
~~Webs, like foam and alien walls and bullshit, are 2d.~~ handled

image
Don't do this (layering issue)

image
Need to work out something for shield walls, either spritewise or on planemaster wise.

image
"Carefully masking???"
image

image
"Bubblegum is getting covered by the lighting on lavaland, but with fullbright, it's fine".

image

"it's technically not broken but I'm redeeming my token for an opinion"

image
Curtins?

### PR DESCRIPTION
**Pull request by ArcaneMusic**
## About The Pull Request

This Pull request performs a resprite of the laser turrets used on station on both the AI satellite, as well as the AI Upload.

https://youtu.be/5NnKZCdUyFE
_Youtube link of this in motion as it was too large for Github natively._

image
image
I don't believe this should have any effect on syndicate rifle turrets, just energy based turrets where applicable.

## Why It's Good For The Game

I made these sprites to help close https://github.com/wall-nerds/wallening/issues/216 originally, as the turret sprites stood out as being particularly flat compared to  many of the other objects, and since I was able to fit it into a 32x32 sprite it felt like something to try and PR downstream beforehand.

In short, it's a 3/4ths turret sprite, at least to the best of my abilities. I still have a handful of sprites to recolor for this related to the CTF variants of the turrets first, so I'm starting this off drafted, but once completed this should be good to go. Looking for feedback as necessary.

## Changelog

:cl:
image: Your station's standard issue E-219 laser turrets within the AI upload and AI satellite have been updated to a newer, shinier model.
/:cl:

### CODE REVIEW

**Comment by ArcaneMusic** [comment]
CTF variants (red/blue) are now finished. Undrafting.
image

**Review by Wallemations — APPROVED** [review]
Shiny

**Comment by Admiral-Gigaton** [comment]
The goon-ifying has arrived, the wallening is coming closer

**Comment by projectkepler-ru** [comment]
THOSE ARE REALLY GOOD ICON ❤️

**Comment by HerCoyote23** [comment]
Imo Stun should have the yellow highlights, as taser bolts are yellow.
Same deal for lethal, think they'd be more clear if they were a red trim, cause they fire red lasers.

There's a precedent with eguns/pulse gun in hands/sprites having the color of shot they're set to.

**Comment by Admiral-Gigaton** [comment]
> Imo Stun should have the yellow highlights, as taser bolts are yellow. Same deal for lethal, think they'd be more clear if they were a red trim, cause they fire red lasers.
> 
> There's a precedent with eguns/pulse gun in hands/sprites having the color of shot they're set to.

You can put any egun type inside, sooo...

Minus magic
